src/SurfaceFluxes.jl

```diff
@@ -196,6 +196,7 @@ Base.@kwdef struct Coefficients{FT, VI, VS} <: AbstractSurfaceConditions{FT, VI,
     z0m::FT
     z0b::FT
     gustiness::FT = FT(1)
+    beta::FT = FT(1)
```

Comment:
Is it necessary to also add this to the `Fluxes` / `FluxesandFrictionVelocity` methods? In the current call to `surface_conditions`, it seems, if `Fluxes` or `FluxesandFrictionVelocity` are known, then:
(1) SHF, LHF are known, [SHF uses the evaporation function, so known SHF implies that the \beta factor, or land surface resistance is known]
(2) Ch is computed from the Monin-Obukhov length 
(3) ` -ρ_sfc * Ch * windspeed(sc) * Δqt(param_set, sc) * sc.beta` still gives the evaporation right ? With some known `beta` dependent on the surface? i.e. here : https://github.com/CliMA/SurfaceFluxes.jl/blob/60fd6e0c01d15c76e2c83c8ce1d73dcf763bcf1d/src/SurfaceFluxes.jl#L306?

Reply:
If `Fluxes` are known then we have https://github.com/CliMA/SurfaceFluxes.jl/blob/60fd6e0c01d15c76e2c83c8ce1d73dcf763bcf1d/src/SurfaceFluxes.jl#L722, but it seems we might still need the `beta` factor for line 306 - is this correct ?

Reply:
Resolved in latest commit ( added dispatch for `evaporation` function given latent heat fluxes)